data/scenarios/03challenges/test.yaml

```diff
@@ -1,20 +1,15 @@
-name: Moving, part 1
+name: Test challenge
```

Comment:
The global description is missing, so nothing is shown in the menu UI.

Reply:
Sure, added a description.